src/core/Track.cpp

```diff
@@ -37,2405 +37,18 @@
 
```

Comment:
I did not see it last review - Can you please fix the top comment (lines 2 and 3) and the doxygen `\file` comment (line 27)?